docs/install.md

```diff
@@ -276,36 +276,73 @@ pmem-deployment      lvm                                 Running  50s
 Annotations:  <none>
 API Version:  pmem-csi.intel.com/v1alpha1
 Kind:         Deployment
 Metadata:
-  Creation Timestamp:  2020-01-23T13:40:32Z
+  Creation Timestamp:  2020-10-07T07:31:58Z
   Generation:          1
-  Resource Version:    3596387
-  Self Link:           /apis/pmem-csi.intel.com/v1alpha1/deployments/pmem-csi.intel.com
-  UID:                 454b5961-5aa2-41c3-b774-29fe932ae236
+  Managed Fields:
+    API Version:  pmem-csi.intel.com/v1alpha1
+    Fields Type:  FieldsV1
+    fieldsV1:
+      f:spec:
+        .:
+        f:deviceMode:
+        f:nodeSelector:
+          .:
+          f:storage:
+    Manager:      kubectl-create
+    Operation:    Update
+    Time:         2020-10-07T07:31:58Z
+    API Version:  pmem-csi.intel.com/v1alpha1
+    Fields Type:  FieldsV1
+    fieldsV1:
+      f:status:
+        .:
+        f:conditions:
+        f:driverComponents:
+        f:lastUpdated:
+        f:phase:
+    Manager:         pmem-csi-operator
+    Operation:       Update
+    Time:            2020-10-07T07:32:22Z
+  Resource Version:  1235740
+  Self Link:         /apis/pmem-csi.intel.com/v1alpha1/deployments/pmem-csi.intel.com
+  UID:               d8635490-53fa-4eec-970d-cd4c76f53b23
 Spec:
-  Controller Resources:
-    Requests:
-      Cpu:      200m
-      Memory:   100Mi
   Device Mode:  lvm
-  Image:        localhost/pmem-csi-driver:canary
-  Node Resources:
-    Requests:
-      Cpu:     200m
-      Memory:  100Mi
+  Node Selector:
+    Storage:  pmem
 Status:
-  Phase:  Running
+  Conditions:
+    Last Update Time:  2020-10-07T07:32:00Z
+    Reason:            Driver certificates are available.
+    Status:            True
+    Type:              CertsReady
+    Last Update Time:  2020-10-07T07:32:02Z
+    Reason:            Driver deployed successfully.
+    Status:            True
+    Type:              DriverDeployed
+  Driver Components:
+    Component:     Controller
+    Last Updated:  2020-10-08T07:45:13Z
+    Reason:        1 instance(s) of controller driver is running successfully
+    Status:        Ready
+    Component:     Node
+    Last Updated:  2020-10-08T07:45:11Z
+    Reason:        All 3 node driver pod(s) running successfully
+    Status:        Ready
+  Last Updated:    2020-10-07T07:32:21Z
+  Phase:           Running
+  Reason:          All driver components are deployed successfully
 Events:
-  Type    Reason         Age                From               Message
-  ----    ------         ----               ----               -------
-  Normal  NewDeployment  34s                pmem-csi-operator  Processing new driver deployment
-  Normal  Running        2s (x10 over 26s)  pmem-csi-operator  Driver deployment successful
-
+  Type    Reason         Age   From               Message
+  ----    ------         ----  ----               -------
+  Normal  NewDeployment  58s   pmem-csi-operator  Processing new driver deployment
+  Normal  Running        39s   pmem-csi-operator  Driver deployment successful
```

Comment:
Looks good. However, the "AGE" field in "kubectl get" is now (?) empty:
```
$ kubectl get deployments.pmem-csi.intel.com
NAME                 DEVICEMODE   NODESELECTOR         IMAGE   STATUS    AGE
pmem-csi.intel.com   lvm          {"storage":"pmem"}           Running   
```

Is that related to this PR?

Reply:
There was a typo in the field name. https://github.com/intel/pmem-csi/pull/749/commits/374202455470d6e1f28eb555855ccf27fdc3d7ab fixes this.

Reply:
Is this resolved?

Reply:
Ah, "operator/crd: Fix typo in additional columns field" was added for it.